docs/experimental/python-provider.md

```diff
@@ -0,0 +1,11 @@
+# Python Provider using Generic Provider
+
+We are using the jedi-language-server (https://github.com/pappasam/jedi-language-server) to make a python provider using the generic provider.
+
+jedi-language-server can be installed using
+
+```
+pip install jedi-langauge-server
+```
+
+It will be installed in `/home/<user_name>/.local/bin/jedi-language-server`
```

Comment:
you could also add the example config/sample main.py/example rule just for the sake of completeness